package.json

```diff
@@ -18,72 +18,84 @@
   "bugs": {
     "url": "https://github.com/getblocklab/block-lab/issues"
   },
-  "dependencies": {
-    "classnames": "^2.2.6"
-  },
   "devDependencies": {
-    "@testing-library/jest-dom": "5.1.1",
-    "@testing-library/react": "9.4.0",
-    "@wordpress/api-fetch": "3.10.0",
-    "@wordpress/babel-preset-default": "4.8.0",
-    "@wordpress/block-editor": "3.4.0",
-    "@wordpress/components": "8.5.0",
-    "@wordpress/compose": "3.9.0",
-    "@wordpress/data": "4.11.0",
-    "@wordpress/dependency-extraction-webpack-plugin": "2.1.0",
-    "@wordpress/dom": "2.8.0",
-    "@wordpress/editor": "9.9.0",
-    "@wordpress/element": "2.10.0",
+    "@testing-library/jest-dom": "5.5.0",
+    "@testing-library/react": "10.0.3",
+    "@wordpress/api-fetch": "3.14.0",
+    "@wordpress/babel-preset-default": "4.12.1",
+    "@wordpress/block-editor": "3.10.0",
+    "@wordpress/components": "9.5.0",
+    "@wordpress/compose": "3.14.0",
+    "@wordpress/core-data": "2.15.0",
+    "@wordpress/data": "4.17.0",
+    "@wordpress/dependency-extraction-webpack-plugin": "2.6.0",
+    "@wordpress/dom": "2.9.0",
+    "@wordpress/e2e-test-utils": "4.6.0",
+    "@wordpress/e2e-tests": "1.15.0",
+    "@wordpress/editor": "9.15.0",
+    "@wordpress/element": "2.13.1",
+    "@wordpress/env": "1.3.0",
```

Comment:
Most of the diff in `package.json` is updating existing packages, or removing the `^` from the version. There are some new packages, though.